src/index.html

```diff
@@ -6,9 +6,30 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a class ="logo" href ="logo"><img src ="/src/images/logo.png" alt = "Logo"></a>
+      <nav class="nav">
+        <ul class = "nav__list">
+          <li class = "nav__item"><a class ="link link_1 is-active" href="/src/index.html">apple</a></li>
```

Comment:
1. link1 is redundant
2. use `nav__link` instead of `link` (to make it as an element of `nav`)
```suggestion
          <li class = "nav__item"><a class ="nav__link is-active" href="/src/index.html">apple</a></li>
```